moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -38,12 +38,19 @@
 
   <class name="default_planner_request_adapters/AddIterativeSplineParameterization" type="default_planner_request_adapters::AddIterativeSplineParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
     <description>
+      A deprecated time parameterization algorithm.
```

Comment:
Off topic and not related to this PR, but can we make a moveit2 issue to remove this algorithm from moveit2?

Reply:
Well, are you sure it should be removed? I googled around and checked out MoveIt1 issues. Did not see any issue reports.

Reply:
I'm unclear if it's IterativeSpline or IterativeParabolic that has major issues. Or maybe both/neither ...?

Reply:
Per this issue from the UR repo, both have issues and probably should be deleted.

https://github.com/ros-planning/moveit/issues/1711

Reply:
Made this issue:  https://github.com/ros-planning/moveit2/issues/573